src/cli/databases_create.rs

```diff
@@ -78,22 +79,40 @@ fn clusters_create(
 
     debug!("Running databases create for {}", &definition);
 
-    let guard = state.lock().unwrap();
+    let guard = &mut state.lock().unwrap();
     let control = if let Some(c) = capella {
         guard.capella_org_for_cluster(c)
     } else {
         guard.active_capella_org()
     }?;
     let client = control.client();
-    let deadline = Instant::now().add(control.timeout());
+    let timeout = control.timeout();
+    let deadline = Instant::now().add(timeout);
 
     let project_name = match control.active_project() {
         Some(p) => p,
         None => {
             return Err(no_active_project_error(span));
         }
     };
-    let project_id = find_project_id(ctrl_c.clone(), project_name, &client, deadline, span)?;
+
+    let org_id = match control.id() {
+        Some(id) => id,
+        None => {
+            let id = find_org_id(ctrl_c.clone(), &client, deadline, span)?;
+            guard.set_active_capella_org_id(id.clone())?;
```

Comment:
Why do we set global state here for a local command specifying the id?

Reply:
So to do anything with projects we need the org_id. Instead of having to fetch it every time we do a command I wanted to save it for future use, so if the org_id is blank then we fetch it and add it to the struct representing the org.